manifests/kots-config.yaml

```diff
@@ -3,297 +3,703 @@ apiVersion: kots.io/v1beta1
-          type: text
-          required: true
-          value: 'https://storage.googleapis.com/{{repl LicenseFieldValue "selfHostedGcpProjectId" }}-thumbnails-storage'
-          default: 'https://storage.googleapis.com/{{repl LicenseFieldValue "selfHostedGcpProjectId" }}-thumbnails-storage'
-        - name: cartoConfigValues_cartoAuth0ClientId
-          title: cartoConfigValues >> cartoAuth0ClientId
-          help_text: "Copy the 'cartoConfigValues.cartoAuth0ClientId.value' yaml"
-          type: text
-          required: true
-          hidden: true
-          default: '{{repl LicenseFieldValue "cartoAuth0ClientId" }}'
-          value: '{{repl LicenseFieldValue "cartoAuth0ClientId" }}'
-        - name: cartoConfigValues_cartoSelfhostedDwLocation
-          title: cartoConfigValues >> cartoSelfhostedDwLocation
-          help_text: "Copy the 'cartoConfigValues.cartoSelfhostedDwLocation.value' yaml"
-          type: text
-          required: true
-          default: '{{repl LicenseFieldValue "cartoSelfhostedDwLocation" }}'
-          value: '{{repl LicenseFieldValue "cartoSelfhostedDwLocation" }}'
-          hidden: true
-        - name: cartoConfigValues_launchDarklyClientSideId
-          title: cartoConfigValues >> launchDarklyClientSideId
-          help_text: "Copy the 'cartoConfigValues.launchDarklyClientSideId.value' yaml"
-          type: text
-          required: true
-          default: '{{repl LicenseFieldValue "launchDarklyClientSideId" }}'
-          value: '{{repl LicenseFieldValue "launchDarklyClientSideId" }}'
-          hidden: true
-        - name: cartoConfigValues_selfHostedGcpProjectId
-          title: cartoConfigValues >> selfHostedGcpProjectId
-          help_text: "Copy the 'cartoConfigValues.selfHostedGcpProjectId.value' yaml"
-          type: text
-          required: true
-          default: '{{repl LicenseFieldValue "selfHostedGcpProjectId" }}'
-          value: '{{repl LicenseFieldValue "selfHostedGcpProjectId" }}'
-          hidden: false
-          readonly: true
-        - name: cartoConfigValues_selfHostedTenantId
-          title: cartoConfigValues >> selfHostedTenantId
-          help_text: "Copy the 'cartoConfigValues.selfHostedTenantId.value' yaml"
-          type: text
-          required: true
-          default: '{{repl LicenseFieldValue "selfHostedTenantId" }}'
-          value: '{{repl LicenseFieldValue "selfHostedTenantId" }}'
-          hidden: false
-          readonly: true
-        - name: cartoConfigValues_customerPackageVersion
-          title: cartoConfigValues >> customerPackageVersion
-          help_text: "Copy the 'cartoConfigValues.customerPackageVersion.value' yaml"
-          type: text
-          required: true
-          default: "null"  # That is going to leave the default provided by Helm
+      - name: storageBucketsBehaviour
+        title: Select your Buckets
+        type: select_one
+        default: default
+        items:
+          - name: default
```

Comment:
That option should be deleted and, for gcs the default values should be our buckets. (Pending to discuss with Alberto if that default should only be present if is a Dev license)